assets/alertmanager/secret.yaml

```diff
@@ -12,8 +12,6 @@ metadata:
   namespace: openshift-monitoring
 stringData:
   alertmanager.yaml: |-
-    "global":
-      "resolve_timeout": "5m"
```

Comment:
this is already the default value from Alertmanager + the setting has no impact in a traditional Prometheus -> Alertmanager workflow (it only applies if the received alerts have no end date).